src/fragment-components/tiles/tile-morphism.tsx

```diff
@@ -113,27 +113,30 @@ export const TileMorphism = ({ component, setComponent }: any) => {
 		{
 			id: 'tile',
 			text: 'Tile',
+			type: 'tile',
```

Comment:
This isn't needed, the type is defined in defaultComponent - this isn't being used anywhere. Can you try removing and try morhping the tiles? It should still work